--- a/maturin/setup.py
+++ b/maturin/setup.py
@@ -65,7 +65,7 @@
     cmdclass={"bdist_wheel": bdist_wheel},
     packages=["maturin"],
     rust_extensions=[
-        RustBin("maturin", args=cargo_args, cargo_manifest_args=["--locked"])
+        RustBin("maturin", args=cargo_args, cargo_manifest_args=["--locked", "--offline"])
     ],
     classifiers=[
         "Topic :: Software Development :: Build Tools",
